src/content/1.7/development/coding-standards/_index.md

```diff
@@ -29,6 +29,15 @@ PHP files MUST follow the [PSR-2 standard](https://www.php-fig.org/psr/psr-2/) a
 Altough [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) are suggested, they are not enforced.
 {{% /notice %}}
 
+[PHP CS Fixer](https://cs.sensiolabs.org/) has been configured for the PrestaShop project to help developers to comply with these conventions.
+
+You can run it using the following command:
+```bash
+php ./vendor/bin/php-cs-fixer fix
+```
+
+And you can also use the provided [git pre-commit](https://github.com/PrestaShop/PrestaShop/tree/develop/.github/contrib) sample in order to make sure you never forget to make your code compliant !
```

Comment:
How about the Composer command which install the git hook? https://github.com/PrestaShop/PrestaShop/blob/develop/composer.json#L176